server/src/utils/upload.ts

```
@@ -63,18 +73,86 @@ const storage = multer.diskStorage({
+      return true;
+    }
+
+    return false;
+  } catch (err) {
+    console.error('Error validating encrypted file:', err);
+    return false;
+  }
+};
+
+// Export the validation function so it can be used in route handlers
+export const validateEncryptedFileAfterUpload = (filePath: string): boolean => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'validateEncryptedFileAfterUpload'. Consider to delete it or rename it.

'validateEncryptedFileAfterUpload' is defined here:


(lint/suspicious/noRedeclare)